transfer/transfer-05-file-transfer-cloud/cloud-transfer-consumer/build.gradle.kts

```diff
@@ -36,6 +36,20 @@ dependencies {
     implementation(libs.edc.dsp)
     
     implementation(libs.edc.data.plane.selector.core)
+
+    //configuration from earlier examples
```

Comment:
this is not "configuration" but these are dependencies, please update the comment accordingly (or removing it altogether, as it is not really necessary). same for the other build file

Reply:
you are right, I just did a minor cleanup 😉